_layouts/page.html

```diff
@@ -0,0 +1,18 @@
+---
+# Forked from the default minima layout to include our main.css
```

Comment:
It already did that on my system, were you having problems?  I don't see any layout-affecting differences when diffing a copy of the `_site` dir generate with this file to one generated without it, nor do I on a visual inspection of the rendered site.

OTOH, even if creating this file now is unnecessary, it's likely something that we'll end up customizing later anyway, so I'm not opposed to this.

Reply:
Oh, I see, the CSS file was moved from its default location in d091a44f7b12b9ab19668112333c193d84af5a4c and only the homepage layout was updated to reflect that, so without this file, the newsletters don't get style changes.

This is fine.  An alternative is to move the css file back to its original location in /assets/ and drop the special code from the home layout.  I don't think it matters long term if this site is continued to be customized.

Reply:
Yup; I prefer to keep `_layout/page.html` bundled as it's convenient to have the code ready at hand for modification, as well as for easier discovery - had to google around a bit to figure out where the page template lived beforehand.